src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -110,6 +113,16 @@ public ResourceTypeSchema(Schema schema) {
             ((ArrayList<?>) v).forEach(p -> this.writeOnlyProperties.add(new JSONPointer(p.toString())));
             return null;
         });
+
+        this.unprocessedProperties.computeIfPresent("handlers", (k, v) -> {
+            ((HashMap<?, ?>) v).keySet().forEach(handlerKey -> {
+                handlerPermissions.put(handlerKey.toString(), new HashSet<>());
+                List<?> permissionsList = (List<?>) ((HashMap<?, ?>) ((HashMap<?, ?>) v).get(handlerKey)).get("permissions");
+                permissionsList.forEach(permission -> handlerPermissions.get(handlerKey.toString()).add(permission.toString()));
+            });
+            // timeoutInMinutes still unprocessed
```

Comment:
Maybe a good way is to model the permissions + timeoutInImutes as a POJO such as "Handler".  Then we have separated method to retrieve them

Reply:
agree with that. i can restructure so this is the case